packages/client/hmi-client/src/views/DataExplorer.vue

```diff
@@ -581,20 +559,16 @@ onUnmounted(() => {
 }
 
 .p-buttonset button {
-	outline: 1px solid var(--text-color-disabled);
-	border: none;
+	border: 1px solid var(--surface-border);
 	box-shadow: none;
-	background-color: var(--surface-a);
-	color: var(--text-color-primary);
-	padding: none;
-	padding: 0.3rem;
+	background-color: var(--surface-0);
 }
 
 .p-buttonset button[active='true'],
 .p-button.p-button-text:enabled:focus {
 	font-weight: bold;
 	border: none;
-	background-color: var(--primary-color-lighter);
+	background-color: var(--surface-highlight);
 }
 
```

Comment:
Is it okay that the selected button doesn't have a border, looks a bit off

Reply:
I'm working on redesigning the buttons at the theme level, we shouldn't have to design them on a view.

Reply:
Agree it looks a bit odd